packages/one-app-bundler/README.md

```diff
@@ -277,6 +277,22 @@ You can pass only one if you wish to customize a single build target.
 
 #### [`purgecss`](https://github.com/FullHuman/purgecss) Options
 
+`purgecss` is an opt-in optimization that can reduce the overall bundle size of your module by
+eliminating unused css from your module's bundle. You can enable `purgecss` by setting
+`bundler.purgecss.enabled` to `false` in the `one-amex` key in your module's `package.json`:
+
+```json
+{
+  "one-amex": {
+    "bundler": {
+      "purgecss": {
+        "disabled": false
```

Comment:
Something is wrong between line 282 and this one.

first mentions `enabled` set to `false` which i think is wrong
second mentions `disabled` set to `false`


What are we going for `enabled set to true` enables. 
or `disabled set to false` enables?

Reply:
oops - `disabled` set to `false` should be the way, since it's backwards-compatible

Reply:
I've made the change but want to keep the discussion open for `disabled` vs `enabled`. `enabled` reads better but is not backwards compatible. `disabled` is backwards-compatible but reads much more awkwardly.

Reply:
I would prefer to have `"enabled": true` as it reads better. we can still support "disabled" but add a deprecation notice.